src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -7,5 +7,6 @@ public enum DocumentFlags
     {
         None = 0,
         Versioned = 1,
+        Artificial = 2,
```

Comment:
It's ok. I was just thinking about a situation 'in the future' when you will have 2 or more places that are producing 'artificial' documents but from one of those you will want to replicate them. If was my only concern.